cmd/skipper/main.go

```diff
@@ -103,6 +109,9 @@ func init() {
 	flag.StringVar(&metricsPrefix, "metrics-prefix", "", metricsPrefixUsage)
 	flag.BoolVar(&debugGcMetrics, "debug-gc-metrics", false, debugGcMetricsUsage)
 	flag.BoolVar(&runtimeMetrics, "runtime-metrics", true, runtimeMetricsUsage)
+	flag.StringVar(&applicationLog, "application-log", "", applicationLogUsage)
+	flag.StringVar(&applicationLogPrefix, "application-log-prefix", "", applicationLogPrefixUsage)
```

Comment:
yup, we'll have it in our own appliance. notice that the default for the access log here is that it is turned off. The reason is that when someone just starts the app in dev mode, may not want to be flooded with access log. And then the prefix is not required. Is this a false logic?